lua/nvim-tree/legacy.lua

```diff
@@ -297,6 +298,99 @@ local function removed(opts)
   end
 end
 
+local OLD_MAPPING_TABLE = {
```

Comment:
We could be lazy and move this to `DEFAULT_KEYMAPS` with a `legacy_action` field.

Reply:
Yes, also i've noticed from some PRs on nvim-treesitter that vim.keymap.set takes an optional "desc" argument which we should use so the nvim-tree will work in which-key and allow us to use that in the help menu